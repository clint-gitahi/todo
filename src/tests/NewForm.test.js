import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import NewForm from '../components/newTodoForm';

test('should render NewForm component without crashing', () => {
  const div = document.createElement('div');

  ReactDOM.render(<NewForm />, div);
});

// test('should handle onSubmit handler', () => {
//   const onSubmitSpy = jest.fn();
//   const wrapper = shallow(<NewForm />);
//   expect(wrapper).toMatchSnapshot();
// });
