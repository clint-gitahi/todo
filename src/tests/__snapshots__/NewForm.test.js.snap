// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should handle onSubmit handler 1`] = `ShallowWrapper {}`;

exports[`should test NewForm component 1`] = `ShallowWrapper {}`;
